meta-lmp-base/recipes-bsp/efitools/efitools_git.bb

```diff
@@ -43,10 +43,15 @@ python do_prepare_local_auths() {
     dir = d.expand('${UEFI_SIGN_KEYDIR}/')
 
     import shutil
+    import os
```

Comment:
typos in the commit message

Reply:
hi daiane, which commit?

Reply:
maybe my github displayed an old version when I commented. It's fixed now